src/sentry/scim/endpoints/members.py

```diff
@@ -437,6 +439,8 @@ def post(self, request: Request, organization) -> Response:
                 )
 
                 # TODO: are invite tokens needed for SAML orgs?
+                member.flags["idp:provisioned"] = True
+                member.flags["idp:role-restricted"] = idp_role_restricted
```

Comment:
@AniketDas-Tekky how do these flags get reset?

Reply:
There's a ticket to handle IDP uninstallation and reset these flags if it happens.